source/manual/cdn.html.md

```diff
@@ -130,14 +130,24 @@ We occasionally decide to ban an IP address at our CDN edge if they exhibit the
 
 Banning IPs shouldn't be taken lightly because many users can share the same IP address and the user behind an IP address can change over time, so there's always a chance that we may block legitimate users.
 
-You can change the list of banned IP addresses by modifying the [YAML config file][ip_ban_config] and [deploying the configuration][ip_ban_deploy].
+You can change the list of banned IP addresses by modifying the [YAML config file][ip_ban_config] and [deploying the configuration][dictionary_deploy].
 
 [ip_ban_config]: https://github.com/alphagov/govuk-cdn-config-secrets/blob/master/fastly/dictionaries/config/ip_address_denylist.yaml
-[ip_ban_deploy]: https://deploy.blue.production.govuk.digital/job/Update_CDN_Dictionaries/build
+
+## Banning requests based on their JA3 signature, at the CDN edge
+
+[JA3 is a way of fingerprinting TLS connections](https://engineering.salesforce.com/open-sourcing-ja3-92c9e53c3c41/), which can be used to detect whether a connection comes from a particular browser, or another TLS client (like curl, python, or possibly malware).
+
+Much like the IP addresses logic above, we're able to block traffic based on its JA3 signature. To do this:
+
+1) Update the [JA3 signature denylist dictionary](https://github.com/alphagov/govuk-cdn-config-secrets/blob/main/fastly/dictionaries/config/ja3_signature_denylist.yaml)
+2) [Deploy the dictionary][dictionary_deploy] to the `www` and `assets` services.
+
+Note that banning JA3s is potentially risky. If we get it wrong, we could ban a legitimate browser version.
```

Comment:
Isn't it also trivial for an attacker to change their fingerprint if they're aware that that's how they're being blocked?